src/ansi-c/cprover_builtin_headers.h

```diff
@@ -10,8 +10,9 @@ __CPROVER_bool __CPROVER_is_invalid_pointer(const void *);
 _Bool __CPROVER_is_zero_string(const void *);
 __CPROVER_size_t __CPROVER_zero_string_length(const void *);
 __CPROVER_size_t __CPROVER_buffer_size(const void *);
-__CPROVER_bool __CPROVER_r_ok(const void *, __CPROVER_size_t);
-__CPROVER_bool __CPROVER_w_ok(const void *, __CPROVER_size_t);
+__CPROVER_bool __CPROVER_r_ok();
```

Comment:
Why did these lose their argument types? Am I missing something?

Reply:
This is the C syntax for "allow any number of arguments".